test/bats/test.bats

```diff
@@ -455,3 +455,27 @@ __expansion_audit_test() {
   run kubectl delete --ignore-not-found -f test/expansion/deployment_no_label.yaml
   run kubectl delete --ignore-not-found -f test/expansion/deployment_with_label.yaml
 }
+
+@test "gatekeeper pubsub test" {
+  if [ -z $ENABLE_PUBSUB_TESTS ]; then
+    skip "skipping pubsub tests"
+  fi
+
+  run kubectl create ns nginx
+  run kubectl create -f test/pubsub/nginx_pod.yaml
+  run kubectl create -f test/pubsub/nginx_pod.yaml
```

Comment:
why are we running this 4 times?

Reply:
To create 4 different pods that are in violation to get multiple messages published and test that we receive all the messages.